src/hooks/data/vaults/fallbacks/apr/useCurveVaultArbitrumTotalApr.ts

```diff
@@ -25,22 +35,30 @@ export default function useCurveVaultArbitrumTotalApr({
   return curveApiQuery
 }
 
+const respSchema = z.object({
+  apys: z.record(
+    z.object({
+      baseApy: z.number(),
+      crvApy: z.number(),
+      cvxApy: z.number(),
+      crvBoost: z.string().or(z.number()),
+      extraRewards: z.array(z.unknown()),
+    })
+  ),
+})
+
 async function getCurveArbitrumApi(poolCurveAddress: Address) {
-  const resp = await axios.get(`${CONVEX_SIDECHAINS_URL}`)
-  const apys = resp?.data?.apys
+  const resp = await axios.get(convexSidechainsUrl)
+  const parsed = respSchema.parse(resp.data)
   let totalApr = 0
-  Object.entries(apys).forEach((entry) => {
-    const key = entry[0]
-    const value: any = entry[1]
+  Object.entries(parsed.apys).forEach(([key, value]) => {
     if (
-      key !== undefined &&
       key.toLocaleLowerCase().includes(poolCurveAddress.toLocaleLowerCase())
     ) {
-      const baseApy = Number(value?.baseApy) / 100
-      const crvApy = Number(value?.crvApy) / 100
-      const cvxApy = Number(value?.cvxApy) / 100
-      const extraRewards = Number(value?.extraRewards)
```

Comment:
`extraRewards` is always an empty array?

Reply:
not sure but I think it's decided by the convex community

Reply:
I need to find a sample response because right now its just an empty array (`Number([]) === 0`). Do you know what it is an array of?